<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_SelectVirtalEnvArea">
    <item row="2" column="0">
     <widget class="QPushButton" name="button_AlreadyLaunched">
      <property name="text">
       <string>[I have already launched AirSim myself]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_VirtualEnvSelectPrompt">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Select A Virtual  Environment To Explore:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_EnvLaunchButtons">
      <item row="0" column="1">
       <widget class="QPushButton" name="button_LaunchTestMap">
        <property name="text">
         <string>Launch Test Map</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="button_LaunchCityMap">
        <property name="text">
         <string>Launch City Map</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_LaunchScreenTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#000000;&quot;&gt;DroneVQA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LaunchScreenDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#1315a6;&quot;&gt;Utilizing Artifical Intelligence For Visual Question Anwering With Sumulated Drones&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>331</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_AirSimLaunchInstruction">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#740000;&quot;&gt;Click &lt;/span&gt;&lt;span style=&quot; font-style:italic; color:#740000;&quot;&gt;&amp;quot;No&amp;quot;&lt;/span&gt;&lt;span style=&quot; color:#740000;&quot;&gt; in AirSim to select quadrotor drone , then click here:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_StartVQA">
      <property name="text">
       <string>Start VQA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
